freetextresponse/public/view.js

```diff
@@ -99,9 +93,20 @@ function FreeTextResponseView(runtime, element) {
         responses.forEach(function(item) {
             html += '<li class="other-student-responses">' + item.answer + '</li>';
         });
+        html = html || '<li class="no-response">No responses to show at this time</li>';
```

Comment:
What about i18n?

Reply:
unortunately that doesn't work in xBlocks yet: https://openedx.atlassian.net/browse/PLAT-1141